tests/modules/withmethods/withmethods.py

```diff
@@ -0,0 +1,32 @@
+from typing import List, Tuple
+from math import pi
+
+from tests.modules import withenum
+from tests import modules
+from tests.modules.withenum import TimeUnit
+
+
+class Coordinates:
+    def __init__(self, x: float, y: float):
+        self.x = x
+        self.y = y
+
+
+class Point:
+    PI: float = pi
+    origin = Coordinates(0, 0)
+
+    @staticmethod
+    def from_values(x: int, y: str, u: float) -> 'Point':
+        return Point(x, y, u)
```

Comment:
the `__init__()` takes only 2 parameters: x and y. I don't know why `u` is passed to the constructor, that would break the call